현정/7주차_1620.java

```diff
@@ -0,0 +1,62 @@
+import java.io.*;
+import java.util.*;
+
+public class BJ_1620 {
+    static int n;
+    static int m;
+
+    //해시맵 두 개 선언
+    // dictionary는 <키 밸류>
+    // dictionaryR은 <밸류 키>
+    static HashMap<Integer, String> dictionary = new HashMap<Integer, String>();
+    static HashMap<String, Integer> dictionaryR = new HashMap<String, Integer>();
```

Comment:
dictionary와 dictionaryR 로 명명하기보단 조금 더 알기 쉬운 이름을 하는게 좋아보이네요

```suggestion
    static HashMap<Integer, String> name_dict = new HashMap<Integer, String>();
    static HashMap<String, Integer> number_dict = new HashMap<String, Integer>();
```

혹은 dict, revesed_dict 이런식으로 명명해도 좋겠네요